source/jormungandr/jormungandr/planner.py

```diff
@@ -57,11 +57,20 @@ def __init__(
         self.isochrone_center = isochrone_center
 
 
+class GraphicalIsochronesParameters(object):
```

Comment:
it is only used in test ? If this is the case, maybe you should move it inside test file directly ?

Reply:
This code will be called later!